lib/pipefy_message/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module PipefyMessage
-  VERSION = "0.1.1"
+  VERSION = "2.0.0"
```

Comment:
What versioning schema is followed?
What's the rationale for bumping from `0.1.1` to `2.0.0`?
Is there a `Changelog` file to track changes?

Reply:
To be honest, we don't follow any standard on that, I just upgraded to version 2 because we made a lot of major changes from the first version (0.0.1) to now.

Also, is worth mentioning that we don't use that version to reference the gem version in other projects like pipefy-core, instead, we use the project release tags from GitHub.

ps: when we move to a package registry we'll need to change that and start to use this version in the correct way.